### PR TITLE
[ci] add internlm2.5 models into testcase

### PR DESCRIPTION
**Pull request by zhulinJulia24**
1. add internlm2.5 models into testcase and oc evaluation configs
2. fix transformers's version in dailytest, cannot do vl quatization on transformers 4.42+
3. fix opencompass's version, because newest oc code need scikit_learn==1.5.0 which need python>=3.9.
4. update vl pipeline testcase for deepseek
5. add prompt length = 1 in benchmark
6. git action is unstable because of https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/ , set env to make action use node16

### CODE REVIEW

File: .github/workflows/daily_ete_test.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ on:
         required: true
         description: 'Dependency packages, you can also set a specific version'
         type: string
-        default: 'packaging transformers_stream_generator transformers datasets matplotlib openai attrdict timm modelscope jmespath'
+        default: 'packaging transformers_stream_generator transformers==4.41.2 datasets matplotlib openai attrdict timm modelscope jmespath'
```
Comment (lvhan028) [inline .github/workflows/daily_ete_test.yml:35]: why use fixed transformers?

Comment (lvhan028) [inline .github/workflows/daily_ete_test.yml:35]: Can we upgrade transformers to `4.42.3`?

File: .github/workflows/evaluate.yml [hunk 2]
```diff
@@ -124,7 +126,7 @@ jobs:
         run: |
           git clone --depth=1 https://github.com/open-compass/opencompass.git
           cd opencompass
-          python3 -m pip install --user -e .
+          python3 -m pip install opencompass==0.2.5
```
Comment (lvhan028) [inline .github/workflows/evaluate.yml:129]: Do we still have to `git clone` opencompass when it is installed through `pip install`?

Comment (lvhan028) [reply .github/workflows/evaluate.yml:129]: Can we remove the version limitation on opencompass?

File: .github/workflows/daily_ete_test.yml [hunk 3]
```diff
@@ -123,7 +123,7 @@ jobs:
         - /usr/share/zoneinfo/Asia/Shanghai:/etc/localtime:ro
     steps:
       - name: Clone repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (zhulinJulia24) [reply .github/workflows/daily_ete_test.yml:126]: > Is there sth wrong with `actions/checkout@v3`?

yes, v3 and v4 is unstable. Change to v2 works. But I cannot search the root reason.
mDCy1wIEkw

File: .github/workflows/daily_ete_test.yml (hunk 1 above)
Comment (zhulinJulia24) [reply .github/workflows/daily_ete_test.yml:35]: > Can we upgrade transformers to `4.42.3`?

4bits quantization of vl model is still fail on 4.42.3 version

File: .github/workflows/evaluate.yml (hunk 2 above)
Comment (zhulinJulia24) [reply .github/workflows/evaluate.yml:129]: > Can we remove the version limitation on opencompass?

OC latest code use scikit_learn==1.5.0 which only support python>=3.9. Test image use python=3.8. So I need to fix the version.

Comment (zhulinJulia24) [reply .github/workflows/evaluate.yml:129]: > Do we still have to `git clone` opencompass when it is installed through `pip install`?

need use model and dateset config files in OC, we still need download opencompass code

File: .github/workflows/daily_ete_test.yml (hunk 1 above)
Comment (lvhan028) [reply .github/workflows/daily_ete_test.yml:35]: @AllentDan may follow this up

File: .github/workflows/evaluate.yml (hunk 2 above)
Comment (lvhan028) [reply .github/workflows/evaluate.yml:129]: I see. I am going to put upgrading python in docker image into my TODO list

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**